### PR TITLE
Amount in project detail page will have company default currency symbol

### PR DESCRIPTION
**Pull request by gowsik-ragunath**
## Notion card

https://www.notion.so/saeloun/Team-member-hourly-rate-currency-bug-080f697e4cef4cab9bbd0f10add3570b

## Summary

On the project detail page, the amount displayed against each team member will have the company's default currency symbol before it.

## Preview

Screenshot 2022-06-28 at 12 18 54 PM

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [X] I have manually tested all workflows
- [X] I have performed a self-review of my own code
- [X] I have added automated tests for my code

### CODE REVIEW

**Review by rohitjoshixyz — COMMENTED** [review]
Question: Why don't we use the `FormatAmountService` here as it is merged now?

**Comment by gowsik-ragunath** [comment]
Yeah @rohitjoshixyz , we can use the `FormatAmountService` here instead of formatting amount in the frontend, will update it shortly.

**Comment by gowsik-ragunath** [comment]
@rohitjoshixyz updated the code, now we are using the service to get the formatted amount.

File: spec/models/project_spec.rb [hunk 1]
```diff
@@ -207,4 +227,8 @@
       end
     end
   end
+
+  def format_amount(currency, amount)
+    FormatAmountService.new(currency, amount).process
+  end
```
Comment (rohitjoshixyz) [inline spec/models/project_spec.rb:233]: Can we reuse the project model's format_amount method here?

**Review by rohitjoshixyz — APPROVED** [review]
LGTM

File: spec/models/project_spec.rb (hunk 1 above)
Comment (gowsik-ragunath) [reply spec/models/project_spec.rb:233]: yeah sure, we will move that method out of private scope and reuse it.

Comment (gowsik-ragunath) [reply spec/models/project_spec.rb]: Updated! reused the method in the project model for formatting the values in test cases.

**Pull request merged**